targets/targets.json

```diff
@@ -3693,6 +3693,41 @@
         "release_versions": ["2", "5"],
         "device_name": "STM32F469NI",
         "bootloader_supported": true
+    },
+	"SDP-K1": {
+        "inherits": ["FAMILY_STM32"],
+        "core": "Cortex-M4",
+		"supported_form_factors": ["ARDUINO"],
```

Comment:
alignment alos here plus other lines like 3708 , please review this edit

Reply:
https://github.com/malavikasajikumar/mbed-os/commit/a8fc16e6de6623bd21753c83d113355c1b67fb4f